clientcore/egress_consumer.go

```diff
@@ -40,8 +42,18 @@ func NewEgressConsumerWebSocket(options *EgressOptions, wg *sync.WaitGroup) *Wor
 			defer cancel()
 
 			// TODO: WSS
+			// Skip certificate verification. This should go away once we have a certificate from a trusted CA instead of our own self-signed one
```

Comment:
Note: Having this here will fail the build for the wasm binary used for the browser widget. The solution is still the same though and it's to use a cert from a trusted CA that the browser recognizes

Reply:
This was just to test the connection to egress server and the only way to unblock, was to temporarily add this change.

Reply:
Okay.  Yeah, let's not merge that to main.  This code is used by the widget IIUC, so you can just build your own local widget with whatever changes you need for dev testing.